meson.build

```diff
@@ -53,6 +71,15 @@ foreach s : schemas
   )
 endforeach
 
+# Compile schemas manually
+glib_compile = find_program('glib-compile-schemas', required: true)
+
+run_command([glib_compile, 'schemas'])
+install_data('schemas/gschemas.compiled', install_dir: schemadir)
```

Comment:
I see you also committed the compiled schemas to git - I would prefer not to.

Reply:
Okay